packages/smpc/.gitignore

```diff
@@ -0,0 +1,54 @@
+# Temporary and binary files
```

Comment:
is this whole file necessary? surely you can just edit the existing gitignore for the project?